### PR TITLE
refactor: [TECH-659] move Top bar actions from Scope Selector

### PR DESCRIPTION
**Pull request by simonadomnisoru**
- Create a new component with the top bar actions logic.
- Use it on the `Enrollment` page
- Clean the code related with the actions form the `ScopeSelector`

### CODE REVIEW

**Review by j-nguyenn — COMMENTED** [review]
Thank you for your great work!

File: src/core_modules/capture-core/components/TopBarActions/TopBarActions.container.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    selectedProgramId,
+    selectedOrgUnitId,
+    isUserInteractionInProgress = false,
+    customActionsOnProgramIdReset = [],
+}: Props) => {
+    const [openStartAgainWarning, setOpenStartAgainWarning] = useState(false);
+    const dispatch = useDispatch();
+    const { reset } = useReset();
+    const { setOrgUnitId } = useSetOrgUnitId();
+    const dontShowWarning = () => !isUserInteractionInProgress;
```
Comment (j-nguyenn) [inline src/core_modules/capture-core/components/TopBarActions/TopBarActions.container.js:33]: hey @simonadomnisoru, why do you invoke a function here? 

nit: I see you only refactor file without changing it, but will we have plan to clean up this later? I think a good pattern for the naming here would be `shouldNotShowWarning`? wdyt?

**Review by j-nguyenn — COMMENTED** [review]
Thank you for your great work!

File: src/core_modules/capture-core/components/TopBarActions/TopBarActions.container.js (hunk 1 above)
Comment (simonadomnisoru) [reply src/core_modules/capture-core/components/TopBarActions/TopBarActions.container.js]: Hey, 
I decided to use the `isUserInteractionInProgress` prop directly without storing it in any new variable. Let me know wdyt. 
Thanks!

**Review by j-nguyenn — APPROVED** [review]
👍

**Review by JoakimSM — APPROVED** [review]
LGTM 👍

**Pull request merged**